framework/include/base/ExecFlagRegistry.h

```diff
@@ -9,22 +9,10 @@
 
 #pragma once
 
-#include <shared_mutex>
-
 #include "ExecFlagEnum.h"
 
-namespace moose
-{
-namespace internal
-{
```

Comment:
I think we could still keep these around so that users are encouraged to use the macros instead of the static method directly